src/containers/Dashboard/Forms/CourseForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from "react";
+import SimpleReactValidator from "simple-react-validator";
+import ReactNotification from "react-notifications-component";
+import "react-notifications-component/dist/theme.css";
+import { addNotification } from "../../../utilities/index";
+import API from "../../../api/index";
+import "./CourseForm.scss";
+
+class CourseForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.validator = new SimpleReactValidator();
+    this.state = {
+      name: "",
+      description: ""
+    };
+    this.handleUserInput = this.handleUserInput.bind(this);
+    this.notificationDOMRef = React.createRef();
+    this.submitForm = this.submitForm.bind(this);
+    this.formRef = null;
+  }
+
+  handleUserInput(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value });
+  }
+  submitForm(e) {
+    e.preventDefault();
+    if (this.validator.allValid()) {
+      var { name, description } = this.state;
+      const data = {
+        name,
+        description
+      };
+      API.courseData(data, result => {
+        if (result.status === "201") {
+          // Form reset
+          this.formRef.reset();
+          addNotification(
+            this.notificationDOMRef,
+            "success",
+            "success",
+            result.message
+          );
+        } else if (
+          result.status === "404" ||
+          result.status === "400" ||
+          result.status === "500"
+        ) {
+          addNotification(
+            this.notificationDOMRef,
+            "Error",
+            "danger",
+            result.status
+          );
+        }
```

Comment:
@Humnaa 
After `else if` you need to handle the `else` case as well.

Reply:
I have written else block